lms/templates/cme_register.html

```diff
@@ -378,78 +363,67 @@ <h3 class="message-title">Welcome ${extauth_id}</h3>
        
         <li class="field" style="display: none;" id="sub-specialty-container">
           <label for="sub_specialty">Sub-Specialty</label>
-          <select id="sub_specialty" name="sub_specialty" class="controller">
+          <select id="sub_specialty" name="sub_specialty">
             <option value="">--</option>
           </select>
          
-          <div class="field controlled" id="sub_specialty_free_container" style="display: none;">
-           <br />
-           <input id="sub_specialty_free" type="text" name="sub_specialty_free" title="Please enter your Sub-Specialty here." placeholder="Please enter your Sub-Specialty here." />
-          </div>
         </li>
        
-        <div id="sub_specialty-Cardiology" class="sub_specialty-container" style="display: none;">
+        <div id="sub_specialty-Cardiovascular_Health" class="sub_specialty-container" style="display: none;">
           <option value="">--</option>
-          %for code, sub_specialty in sub_specialty_choices['Cardiology']:
+          %for code, sub_specialty in sub_specialty_choices['Cardiovascular_Health']:
             <option value="${code}">${sub_specialty}</option>
           %endfor
         </div>
-       
+
         <div id="sub_specialty-Internal_Medicine" class="sub_specialty-container" style="display: none;">
           <option value="">--</option>
           %for code, sub_specialty in sub_specialty_choices['Internal_Medicine']:
             <option value="${code}">${sub_specialty}</option>
           %endfor
         </div>
-       
+
         <div id="sub_specialty-Obstetrics_Gynecology" class="sub_specialty-container" style="display: none;">
           <option value="">--</option>
-          %for code, sub_specialty in sub_specialty_choices['Obstetrics_Gynecology']:
+          %for code, sub_specialty in sub_specialty_choices['Obstetrics_&_Gynecology']:
```

Comment:
Actually, do we even need to include the ampersands in the key name?
We're not displaying this literally anywhere, are we?

If we aren't using this directly for the display name, it may make sense to name the keys without special characters; it's one less thing we have to special-case in the Javascript.